buyer-api/src/facades/helpers/performTransaction.js

```diff
@@ -1,4 +1,4 @@
-import { delay } from '../../utils';
+import { delay } from '../../utils/delay';
```

Comment:
Why should we use the absolute path and not the one in index?

Reply:
Yes, it makes no sense, I'll revert it!